src/xtgeo/grid3d/grid.py

```diff
@@ -1559,6 +1564,62 @@ def get_zoneprop_from_subgrids(self) -> NoReturn:
         """Make a XTGeo GridProperty instance for a Zone property subgrid index."""
         raise NotImplementedError("Not yet; todo")
 
+    def get_boundary_polygons(
+        self: Grid,
+        alpha_factor: Optional[float] = 1.0,
+        convex: Optional[bool] = False,
+        simplify: Optional[bool] = True,
```

Comment:
Who do these values need to be optional, seeing that you have added defaults it seems unnecessary.

Reply:
Very true 👍